src/TechBlog/Project.hs

```diff
@@ -63,6 +63,12 @@ allProjects = traverse makeItem
         , pImg  = "postgresql-simple-named.png"
         , pDesc = "Introducing named parameters for the postgresql-simple library"
         }
+    , Project
+        { pName = "Servant Docs Simple"
+        , pLink = "servant-docs-simple"
+        , pImg  = "servant_docs_simple.png"
```

Comment:
You'll need to add this file too if you want it show up correctly :)
The illustrations for the other projects come from Yunus, you can ask Thriguna how to submit a proper requests for the illustration ([form here](https://form.asana.com/?hash=aabd491281d1de3f2768ca14f347b2b995935b736754e5ce360119c588224268&id=1158461494497831))
Usually he'll need a couple of sentences on what the project is about, so he has something to work with for the illustration.

Reply:
I think there is already a image for this project that can be included

Reply:
Do you know who has it?

Reply:
The file is in the commit, I think it is minimized because it is a binary file https://github.com/Holmusk/holmusk.github.io/blob/2688150f09880f2f4ec31d0ea35a550216c1c912/images/projects/servant_docs_simple.png

Reply:
Ah! I completely missed it sorry!